pylint/utils/utils.py

```diff
@@ -311,7 +311,7 @@ def _comment(string):
 def _format_option_value(optdict, value):
     """return the user input's value from a 'compiled' value"""
     if optdict.get("type", None) == "py_version":
-        value = ".".join(str(item) for item in value)
+        value = "current Python interpreter version (e.g., '3.8')"
```

Comment:
Shouldn't this also return the current value specified by the user? At least that's what the docstring says and what the previous version did.

Reply:
```suggestion
        value = f"current Python interpreter version (e.g., '{'.'.join(str(item) for item in value)}')"
```

Reply:
> Shouldn't this also return the current value specified by the user? At least that's what the docstring says and what the previous version did.

Yeah 😅
I didn't though about the help text which also uses that method. Thanks for the pointer. Should be fixed now.